gazebo/sensors/Sensor.cc

```diff
@@ -174,8 +174,18 @@ void Sensor::Update(const bool _force)
   {
     if (this->useStrictRate)
     {
+      // rendering sensors (IMAGE category) has it's own mechanism
+      // for throttling and lockstepping with physics. So throttle just
+      // physics sensors
+      if (this->dataPtr->category != IMAGE && !this->NeedsUpdate() && !_force)
+        return;
+
       if (this->UpdateImpl(_force))
+      {
+        common::Time simTime = this->world->SimTime();
+        this->lastUpdateTime = simTime;
```

Comment:
Nit: could avoid the temporary object here:

```suggestion
        this->lastUpdateTime = this->world->SimTime();
```

Reply:
done. 4ab78c5